Sources/SwerverCore/Server/ClientRequestHandler.swift

```diff
@@ -10,18 +10,18 @@ public class ClientRequestHandler {
     
     public func handle(client: Socket) throws {
         let httpRequest = try parseRequest(clientSocket: client)
-        try client.write(from: httpRouteProcessor.process(request: httpRequest))
+        try client.write(from: httpRouteProcessor.process(request: (httpRequest)))
     }
     
-    private func parseRequest(clientSocket: Socket) throws -> HttpRequest {
-        var readData: Data? = Data(capacity: Swerver.bufferSize)
-        let _ = try clientSocket.read(into: &readData!)
-        guard let parsedData = String(data: readData!, encoding: .utf8) else {
-            readData!.count = 0
+    private func parseRequest(clientSocket: Socket) throws -> HttpRequest? {
+        var readData: Data = Data(capacity: Swerver.bufferSize)
+        let _ = try clientSocket.read(into: &readData)
```

Comment:
Can `let _ = ` be removed here?

Reply:
will look into that

Reply:
if I remove it, the compiler will throw a warning. I believe .read returns a Data object

Reply:
:+1:
Weird code is slightly preferable to a compiler warning.

Reply:
👍